src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -725,35 +737,11 @@ private void onRecentlyPlayedGameEnded(GameBean game) {
     notificationService.addNotification(new PersistentNotification(i18n.get("game.ended", game.getTitle()), Severity.INFO, singletonList(new Action(i18n.get("game.rate"), actionEvent -> eventBus.post(new ShowReplayEvent(game.getId()))))));
   }
 
-  @VisibleForTesting
-  CompletableFuture<Void> spawnTerminationListener(Process process) {
-    return spawnTerminationListener(process, true);
-  }
```

Comment:
Why was this removed? It didn't need to be removed and allows easier method reference.

Reply:
It made it harder for me to find all usages of termination listeners and made the code harder for me to understand. So I refactored it to make it more explicit